Testreglar/252/Nett/nett-2.5.2a.json

```diff
@@ -0,0 +1,189 @@
+{
+	"id": "nett-2.5.2a",
+	"type": "web",
+	"kravTilSamsvar": "<p>For funksjonalitet som kan betjenes med en enkel pekerbevegelse skal minst én av følgende være oppfylt:</p>\r\n<ul>\r\n<li>Trykkhandlingen (down-event) fra brukeren utløser ikke funksjonalitet eller</li>\r\n<li>Funksjonen utføres på slipphandlingen (up-event), og det finnes en mekanisme\r\n<ul>\r\n<li>for å avbryte funksjonen før den utføres, eller</li>\r\n<li>for å angre funksjonen etter at den er utført eller </li>\r\n</ul>\r\n</li>\r\n<li>Slipphandlingen (up-event) opphever resultatet av den foregående trykkhandlingen (down-event) eller</li>\r\n<li>Det er avgjørende å utføre funksjonen på trykkhandlingen (down-event)</li>\r\n</ul>",
+	"kolonner": [
+		{
+			"title": "2.2"
+		},
+		{
+			"title": "3.2"
+		},
+		{
+			"title": "3.3"
+		},
+		{
+			"title": "3.4"
+		},
+		{
+			"title": "3.5"
+		},
+		{
+			"title": "3.6"
+		},
+		{
+			"title": "3.7"
+		}
+	],
+	"side": "2.1",
+	"element": "3.1",
+	"steg": [
+		{
+			"stegnr": "2.1",
+			"spm": "Hvilken side tester du?",
+			"ht": "<p>Angi URL eller side-ID.</p>",
+			"type": "tekst",
+			"kilde": [],
+			"label": "URL/Side-ID:",
+			"datalist": "Sideutvalg",
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "2.2"
+				}
+			}
+		},
+		{
+			"stegnr": "2.2",
+			"spm": "Finnes det funksjonalitet som kan betjenes med en enkel pekerbevegelse (single pointer)?",
+			"ht": "<p>Enkel pekerbevegelse er pekerinndata som fungerer med kun ett kontaktpunkt med skjermen, herunder:</p>\n<ul>\n<li>enkeltrykk og -klikk,</li>\n<li>dobbelttrykk og -klikk,</li>\n<li>lange trykk</li>\n</ul>\n<p>Slik identifiserer du funksjonalitet som kan betjenes med en enkel pekerbevegelse:</p>\n<ul>\n<li>Åpne testsiden på en enhet med berøringsskjerm f.eks. smarttelefon eller nettbrett. </li>\n<li>Slå av skjermleser og andre andre tilgjengelighetsfunksjoner, hvis det er aktivert på enheten\n<ul>\n<li>Dette gjelder både hjelpemiddelteknologi og innstillinger i operativsystemet</li>\n</ul>\n</li>\n<li>Gjør en visuell inspeksjon og sjekk om det finnes lenker og knapper eller lignende funksjonalitet </li>\n</ul>",
```

Comment:
Hvorfor skal man teste på touchskærm? 
Er det ikke også relevant for museklik? (i linje 81 står der "pekeren du bruker (museknappen, fingeren eller stylus)"). 

Der kan vel godt være forskel på touchevents og muse-events? 
Så egentlig bør begge vel testes, ideelt set?

Reply:
"Lenker og knapper eller lignende funktionalitet" --> "interaktive elementer, såsom lenker og knapper"?